<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigsAnimations</class>
 <widget class="QWidget" name="ConfigsAnimations">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" rowspan="3">
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Nome:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QGroupBox" name="groupBoxDadosFrames">
     <property name="title">
      <string>Dados de frames</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Numero de frames na horizontal:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="spinBoxHorizontalFramesNumber">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Numero de frames na vertical:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSpinBox" name="spinBoxVerticalFrameNumber">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="2">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Espaçamento horizontal:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSpinBox" name="spinBoxHorizontalSpacing"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>Espaçamento vertical:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QSpinBox" name="spinBoxVerticalSpacing"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Velocidade:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QSlider" name="horizontalSliderVelocity">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>Loop</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Frame loop:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QSpinBox" name="spinBoxFrameLoop"/>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Ordem:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="4">
       <widget class="QComboBox" name="comboBoxOrder">
        <item>
         <property name="text">
          <string>Esquerda para direita primeiro</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Cima para baixo primeiro</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>Coluna inicial:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBoxColunaInicial">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Coluna final:</string>
        </property>
       </widget>
      </item>
      <item row="8" column="4">
       <widget class="QSpinBox" name="spinBoxColunaFinal">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Linha inicial:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QSpinBox" name="spinBoxLinhaInicial">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="9" column="3">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Linha final:</string>
        </property>
       </widget>
      </item>
      <item row="9" column="4">
       <widget class="QSpinBox" name="spinBoxLinhaFinal">
        <property name="minimum">
         <number>1</number>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>Cor transparência:</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QGraphicsView" name="graphicsViewColorkey">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="10" column="2">
       <widget class="QToolButton" name="toolButtonColorkey">
        <property name="text">
         <string>Escolher</string>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QToolButton" name="toolButtonColorkeyX">
        <property name="text">
         <string>X</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="2" rowspan="2">
    <widget class="QGroupBox" name="groupBoxVisualizacao">
     <property name="title">
      <string>Visualizacao:</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QGraphicsView" name="graphicsView">
        <property name="maximumSize">
         <size>
          <width>999</width>
          <height>300</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="pushButtonAddAnimation">
     <property name="text">
      <string>Adiciona Animação</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
